spec/system/candidate_interface/candidate_entering_personal_details_spec.rb

```diff
@@ -1,41 +1,37 @@
 require 'rails_helper'
 
 RSpec.feature 'Entering their personal details' do
-  scenario 'Logged in candidate with no personal details' do
+  scenario 'Candidate can submit their personal details' do
     given_i_am_signed_in
     and_i_visit_the_site
+
     when_i_click_on_personal_details
-    then_i_can_edit_my_personal_details
-  end
+    and_i_fill_in_some_details_but_omit_some_required_details
+    and_i_submit_the_form
+    then_i_should_see_validation_errors
 
-  scenario 'Candidate fills in personal details' do
-    given_i_am_on_the_personal_details_page
-    when_i_fill_in_the_form
+    when_i_fill_in_the_rest_of_my_details
+    and_i_submit_the_form
     then_i_can_check_my_answers
-  end
 
-  scenario 'Candidate goes back to their application' do
-    given_i_am_on_the_personal_details_page
-    when_i_click_on_back_to_application
-    then_i_can_see_my_application
-  end
-
-  scenario 'Candidate triggers validation errors' do
-    given_i_am_on_the_personal_details_page
-    when_i_submit_the_form
-    then_i_should_see_validation_errors
+    # when_i_click_to_change_my_answer
+    # and_i_fill_in_a_different_answer
+    # and_i_submit_the_form
+    # then_i_can_check_my_revised_answers
+    #
+    # when_i_submit_my_details
+    # then_i_should_see_the_form
+    # and_that_the_section_is_completed
+    #
+    # when_i_click_on_personal_details
+    # then_i_can_check_my_revised_answers
```

Comment:
do we want to keep these comments?

Reply:
Yes, we're working through them with follow-up PRs 👍